src/tree/mod.rs

```diff
@@ -10,15 +10,14 @@
 //!
 //! ### Example
 //!
-//! ```rust
+//! ```rust,no_run
```

Comment:
Why documentation tests are removed? Its important to keep its consistency

Reply:
They are not removed. This flag makes forces the compiler to compile the tests but not run them. And considering that what it done in the test takes quite some time, I think it's Ok to just see if it compiles but not run it.